src/components/auth/ConnectStellarWallet.tsx

```diff
@@ -35,6 +37,14 @@ export const ConnectStellarWallet = ({
       onClick={async (e: any) => {
         kit.setWallet(ALBEDO_ID);
         const { address } = await kit.getAddress();
+        const isFunded = await isFundedStellarWallet(address);
+        if (!isFunded) {
+          toast.error(
+            "Can't find your wallet registry, make sure you're trying to connect a funded wallet"
```

Comment:
I wonder if this is clear for any user / visitor.

Maybe, a text like "Can't find your wallet registry, make sure you're trying to connect a wallet that has XLM funds"

What do you think?

Reply:
If you fund a wallet and then transfer all of your XLM funds, you will have 0 XLM, but the wallet will have a registry and will be valid, but maybe there is a better word than "funded" for that, maybe "initialized"?

Reply:
Replaced the error message for a new one, let me know what you think